src/scripts/content/teamwork.js

```diff
@@ -86,7 +86,7 @@ togglbutton.render('div.taskRHS:not(.toggl), div.row-rightElements:not(.toggl)',
 // Tasks Detail View Page
 togglbutton.render('div#Task div.titleHolder ul.options:not(.toggl), .view-header ul.task-details-options:not(.toggl)', {observe: true}, function (elem) {
```

Comment:
Can you confirm if this whole section is still needed? I think I got to the "Task Detail View Page" but this selector doesn't trigger. Only the selector on "Tasks listing page in project" triggers again, so perhaps both pages can use the same selector/code? It seems to be working.